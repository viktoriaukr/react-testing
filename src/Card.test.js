import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

test("smoke test", () => {
  render(
    <Card
      caption="Photo by Richard Pasquarella on Unsplash"
      src={`./image1.jpg`}
    />
  );
});

test("snapshot test", () => {
  const { asFragment } = render(
    <Card
      caption="Photo by Richard Pasquarella on Unsplash"
      src={`./image1.jpg`}
    />
  );
  expect(asFragment()).toMatchSnapshot();
});

