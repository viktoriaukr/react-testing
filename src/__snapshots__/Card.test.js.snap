// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Photo by Richard Pasquarella on Unsplash
    </h4>
    <img
      alt="Photo by Richard Pasquarella on Unsplash"
      class="Card-image"
      src="./image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
